src/code/ucode_disas.c

```diff
@@ -69,7 +69,12 @@ F3dzexFlag sUCodeDisasMtxFlags[] = {
     F3DZEX_FLAG(G_MTX_PUSH, G_MTX_NOPUSH),
 };
 
-typedef enum { COMBINER_A = 1, COMBINER_B, COMBINER_C, COMBINER_D } CombinerArg;
+typedef enum {
+    COMBINER_A = 1,
+    COMBINER_B,
+    COMBINER_C,
+    COMBINER_D
+} CombinerArg;
```

Comment:
I guess following our current style these technically should have offset comments lol. but I dont find them that useful for cases like this personally

Reply:
in many cases the offset comments don't really have any use and we have them more to force the one-value-per-line formatting than for usefulness

Reply:
very true. maybe we can start removing them eventually 😄 
theres also ofc struct comments that are going to be out of date between versions... whole other conversation though